### PR TITLE
sourcegraph: release 0.4.0

### PR DESCRIPTION
**Pull request by michaellzc**
I backfilled our changelog with commits that I think it's worth mentioning ` git log --oneline -- charts/sourcegraph/`

## Test plan

N/A, new release

### CODE REVIEW

File: charts/sourcegraph/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Chart for installing Sourcegraph
 type: application
 
 # Chart version, separate from Sourcegraph
-version: 0.3.0
+version: 1.0.0
```
Comment (caugustus-sourcegraph) [inline charts/sourcegraph/Chart.yaml:8]: I'm not sure we're ready to 1.0 yet! Maybe just bump to `0.4.0` and we can 1.0 as part of going GA?

Comment (michaellzc) [inline charts/sourcegraph/Chart.yaml:8]:
```suggestion
version: 0.4.0
```

Comment (caugustus-sourcegraph) [reply charts/sourcegraph/Chart.yaml]: I know that's not following strict semver, but 1.0 is special 😄

**Review by caugustus-sourcegraph — APPROVED** [review]

File: charts/sourcegraph/CHANGELOG.md [hunk 2]
```diff
@@ -25,3 +25,23 @@ All notable changes to Sourcegraph are documented in this file.
 ### Removed
 
 -
+
+## 1.0.0
```
Comment (michaellzc) [inline charts/sourcegraph/CHANGELOG.md:29]:
```suggestion
## 0.4.0
```

**Pull request merged**